### DISCUSSION TITLE
Gitea: onboardingConfigFileName in .gitea Directory Not Supported

### DISCUSSION THREAD

**Discussion by mturekk** [renovatebot/renovate#31673] [Suggest an Idea]
### Tell us more.

Should support for storing the Renovate repo configuration file in `.gitea/renovate.json` be supported the way it is for GitHub and GitLab?

This is [not currently supported](https://docs.renovatebot.com/configuration-options/), and attempts to set `onboardingConfigFileName: ".gitea/renovate.json"` results in the following configuration validation warning

```text
WARN: Config validation warnings found in config.js
       "warnings": [
         {
           "topic": "Configuration Error",
           "message": "Invalid value `.gitea/renovate.json` for `onboardingConfigFileName`. The allowed values are renovate.json, renovate.json5, .github/renovate.json, .github/renovate.json5, .gitlab/renovate.json, .gitlab/renovate.json5, .renovaterc, .renovaterc.json, .renovaterc.json5, package.json."
         }
       ]
```

**Comment by RahulGautamSingh** [comment]
I think we can support this. @viceice WDYT??

**Reply by viceice** [reply]
sure, but we need to refactor our code before. we should not support `.gitea/xxx` on `github` .....

So only `.<platform>/renovate.json` should be supported so this:
https://github.com/renovatebot/renovate/blob/75b29f165a34367536f8fef2bb195491cb3ff2f1/lib/config/app-strings.ts#L4-L7

needs to be replaced by a generic platform support.

**Reply by jvllmr** [reply]
Is there an issue ticket for this now? 

I agree that it would be better to only use the folder on the correct platform, but what about adding the .gitea paths to the array until the refactor so people can start declutter their root path asap?
